Build/PackageFiles/Dotnet_precompiled/ItemTemplates-Isolated-NetFx_v4.x.nuspec

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.Azure.Functions.Worker.NetFx.ItemTemplates</id>
+    <version>1.0.$patchVersion$</version>
+    <authors>Microsoft</authors>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=2028464</licenseUrl>
+    <repository type="git" url="https://github.com/Azure/azure-functions-templates" />
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=848145</projectUrl>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Collection of templates for Azure Functions .NET Isolated using .NET Framework</description>
+    <language>en-US</language>
+    <packageTypes>
+      <packageType name="Template" />
+    </packageTypes>
+  </metadata>
+  <files>
+    <file src="..\..\..\Functions.Templates\Templates\QueueTrigger-CSharp-Isolated\**" target="content\QueueTrigger-CSharp\QueueTriggerCSharp.cs" />
+    <file src="..\..\..\Functions.Templates\Templates\QueueTrigger-FSharp-Isolated\**" target="content\QueueTrigger-FSharp\QueueTriggerFSharp.fs" />
+    <file src="..\..\..\Functions.Templates\Templates\HttpTrigger-CSharp-Isolated-NetFx\**" target="content\HttpTrigger-CSharp\HttpTriggerCSharp.cs" />
+    <file src="..\..\..\Functions.Templates\Templates\HttpTrigger-FSharp-Isolated\**" target="content\HttpTrigger-FSharp\HttpTriggerFSharp.fs" />
```

Comment:
`HttpTrigger-FSharp-Isolated` theoretically would also need to be updated, but I suppose that depends on what happens with the project template per another PR discussion. I'm ok keeping this out of the scope of this one, but I just wanted to note it.